exercises/practice/grep/.meta/config.json

```
@@ -0,0 +1,19 @@
+{
+  "authors": [
+    "ryanplusplus"
+  ],
+  "files": {
+    "solution": [
+      "grep.lua"
+    ],
+    "test": [
+      "grep_spec.lua"
+    ],
+    "example": [
+      ".meta/example.lua"
+    ]
```

```suggestion
    "example": [
      ".meta/example.lua"
    ],
    "editor": [
      "iliad.txt",
      "midsummer-night.txt",
      "paradise-lost.txt"
    ]
```

The `.files.editor` section allows files to show up in the web editor, but read-only. People will want to see the files they are grepping.

---

Great catch, thank you!